tests/tests.go

```diff
@@ -80,7 +80,8 @@ agent:
   listen: {{.AgentAddr}}:{{.AgentPort}}
   topology:
     probes:
-    - ovsdb
+	- ovsdb
```

Comment:
Please fix indentation (spaces instead of tabs)

Reply:
oops.  will fix.